hail/python/hail/docs/_static/annotationdb/annotationdb.js

```diff
@@ -0,0 +1,79 @@
+
+
+//CheckAll functionality for UI
+$("#checkAll").click(function(){
+    $('input:checkbox').not(this).prop('checked', this.checked);
+});
+
+//Get and append JSON information to Table
+$.getJSON("https://api.myjson.com/bins/1fn7f7",
```

Comment:
This needs to be changed to the google storage location before we can merge this PR. Let's get the JSON up into google storage today.